src/biz/metrics_aws_test.go

```diff
@@ -118,3 +118,11 @@ func startMetadataEndpointStub(t *testing.T, taskID string) (server *httptest.Se
 	))
 	return ts, ts.Close
 }
+
+func assertBlkIOMetric(t *testing.T, expected float64, actual *float64) {
+	if actual != nil {
+		assert.Equal(t, expected, *actual)
+	} else {
+		assert.Nil(t, actual)
```

Comment:
would make sense depending on the test calling directly 

assert.Equal(t, expected, *actual)

or 
		assert.Nil(t, actual)
?

Reply:
Done!